docs/reference/ilm/getting-started-slm.asciidoc

```diff
@@ -0,0 +1,173 @@
+[role="xpack"]
+[testenv="basic"]
+[[getting-started-snapshot-lifecycle-management]]
+== Getting started with snapshot lifecycle management
+
+Let's get started with snapshot lifecycle management (SLM) by working through a
+hands-on scenario. The goal of this example is to automatically back up {es}
+indices using the <<modules-snapshots,snapshots>> every day at a particular
+time.
+
+[float]
+[[slm-gs-create-policy]]
+=== Setting up a repository
+
+Before we can set up an SLM policy, we'll need to set up a
+<<snapshots-repositories,snapshot repository>> where the snapshots will be
+stored. Repositories can use {plugins}/repository.html[many different backends],
+including cloud storage providers. You'll probably want to use one of these in
+production, but for this example we'll use a shared file system repository:
+
+[source,js]
+-----------------------------------
+PUT /_snapshot/my_repository
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+-----------------------------------
+// CONSOLE
+// TEST
+
+[float]
+=== Setting up a policy
+
+Now that we have a repository in place, we can create a policy to automatically
+take snapshots. Policies are written in JSON and will define when to take
+snapshots, what the snapshots should be named, and which indices should be
+included, among other things.  We'll use the <<slm-api-put,Put Policy>> API
+to create the policy.
+
+[source,js]
+--------------------------------------------------
+PUT /_slm/policy/nightly-snapshots
+{
+  "schedule": "0 30 1 * * ?", <1>
+  "name": "<nightly-snap-{now/d}>", <2>
+  "repository": "my_repository", <3>
+  "config": { <4>
+    "indices": ["*"] <5>
+  }
+}
+--------------------------------------------------
+// CONSOLE
+// TEST[continued]
+<1> when the snapshot should be taken, using 
+    {xpack-ref}/trigger-schedule.html#schedule-cron[Cron syntax], in this 
+    case at 1:30AM each day
+<2> whe name each snapshot should be given, using 
+    <<date-math-index-names,date math>> to include the current date in the name
+    of the snapshot
+<3> the repository the snapshot should be stored in
+<4> the configuration to be used for the snapshot requests (see below)
+<5> which indices should be included in the snapshot, in this case, every index
+
+This policy will take a snapshot of every index each day at 1:30AM UTC //TODO is
+it actually UTC?//.  Snapshots are incremental, allowing frequent snapshots
+to be stored efficiently, so don't be afraid to configure a policy to take
+frequent snapshots.
+
+In addition to specifying the indices that should be included in the snapshot,
+the `config` field can be used to customize other aspects of the snapshot. You
+can use any option allowed in <<snapshots-take-snapshot,a regular snapshot 
+request>>, so you can specify whether the snapshot should fail in special cases,
```

Comment:
"so you can specify" -> "so, for example, you can specify" since we're not exhaustively listing the options here